cumulusci/tasks/robotframework/robotdoc.py

```diff
@@ -0,0 +1,98 @@
+import os
+import os.path
+import time
+import jinja2
+import robot.utils
+
+import cumulusci
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.utils import process_list_arg
+
+from robot.libdocpkg import LibraryDocumentation
+from robot.libraries.BuiltIn import RobotNotRunningError
+
+
+class RobotDoc(BaseTask):
+    task_options = {
+        "outputdir": {
+            "description": "The path to a folder where generated files will be placed",
+            "required": True,
+        },
+        "files": {
+            "description": "A list of keyword definition files (eg: .py, .robot, .resource)",
+            "required": True,
+        },
+    }
+
+    def _validate_options(self):
+        super(RobotDoc, self)._validate_options()
+
+        self.options["files"] = process_list_arg(self.options["files"])
+        if not os.path.exists(self.options["outputdir"]):
+            os.mkdir(self.options["outputdir"])
```

Comment:
Use `os.makedirs` to handle if there are multiple levels of directories.

Reply:
Good point. I was also not sure if I should auto-create it or not. Autocreate works great in the normal case, but it could be annoying if you typo'd the path in the config file and it created some random directory somewhere that you had to clean up.

Reply:
Good point.

Reply:
I switched to using os.makedirs.